package.json

```diff
@@ -122,6 +123,7 @@
     "tincanjs": "^0.50.0",
     "ts-toolbelt": "^9.6.0",
     "video.js": "^7.11.8",
+    "xdm": "^1.11.0",
```

Comment:
Shouldn't need to depend on this directly. It's brought in automatically by mdx-bundler.